app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java

```diff
@@ -308,17 +321,25 @@ public void run() {
     }
 
     @Subscribe
-    public synchronized void updateConcentration(MuseHandler.FocusReading reading) {
-        final int[] rgbColor = mColorMap.iMap(reading.getFocus());
+    public synchronized void updateConcentration(final MuseHandler.FocusReading reading) {
```

Comment:
This doesn't need to be synchronized if all it does is to queue an operation in the UI run loop.